packages/component-lib/src/components/tableList/transactionsTable/TransactionTable.tsx

```diff
@@ -347,9 +350,9 @@ export const TransactionTable = withTranslation(['tables', 'common'])((props: Tr
         {
             key: 'status',
             name: t('labelTxMemo'),
-            headerCellClass: 'textAlignRight',
+            headerCellClass: 'textAlingCenter',
```

Comment:
Typo.
```suggestion
            headerCellClass: 'textAlignCenter',
```

Reply:
the column is alignCenter or not if it's AlignRight keep it as Right

Reply:
He is referring to the typo "Aling", correcting with "Align" instead.